cl/lib/model_helpers.py

```diff
@@ -12,6 +12,41 @@
 from cl.lib.string_utils import normalize_dashes, trunc
 
 
+def clean_docket_number(docket_number: str | None) -> str:
```

Comment:
Should we normalize case and dashes in here too, instead of doing it in two steps?

Maybe shift everything to uppercase or lowercase?

Reply:
Yeah, I wasn't considering a docket number having different case on different sources, so a new docket was going to be created, now normalizing to lowercase solved the problem, added a new test case about this one.

Also added the dashes normalize method inside `clean_docket_number`.

Thanks.